scenarios/mqtt-pravega-bridge/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.4'
+}
+
+group 'com.dell.mqtt'
+version = samplesVersion
+
+task wrapper(type: Wrapper) {
```

Comment:
can it not use the gradle wrapper defined at the root of the samples project?

Reply:
Fixed